cpp/ycm/ClangCompleter/TranslationUnit.cpp

```diff
@@ -389,6 +388,68 @@ void TranslationUnit::UpdateLatestDiagnostics() {
   }
 }
 
+namespace {
+  /// Sort a FixIt container by its location's distance from a given column
+  /// (such as the cursor location)
+  ///
+  /// PreCondition: all FixIts in the container are on the same line
+  struct sort_by_location {
+    sort_by_location( int column )
+      : column_(column) {
```

Comment:
You can write this as `sort_by_location( int column ) : column_( column ) {}`.

Unless the formatter is making it this way. If so, leave it.